assets/js/components/VisuallyHidden.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Visually Hidden component, mainly used for screen-reader text.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+function VisuallyHidden( { className, children, ...otherProps } ) {
+	return <span { ...otherProps } className={ classnames( 'screen-reader-text', className ) }>{ children }</span>;
```

Comment:
I realize this is how it was in the IB, but let's format this a little differently for consistency with other components. No reason to cram this all on to one line 😄 

```suggestion
	return (
		<span
			{ ...otherProps }
			className={ classnames( 'screen-reader-text', className ) }
		>
			{ children }
		</span>
	);
```